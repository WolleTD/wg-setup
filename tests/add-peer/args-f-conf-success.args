-f /etc/wireguard/wg0.conf peer2 iCJSbobpCRHCG2/WP9D1/viGlv+WrNpWtd1XkRzyrFs= 172.16.1.2/32
