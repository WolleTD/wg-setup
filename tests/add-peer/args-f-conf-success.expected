# wg0

\[Peer\]
# peer2
# Added by [[:alnum:]_-]+ at [0-9]{4}-[0-9]{2}-[0-9]{2}
PublicKey = iCJSbobpCRHCG2/WP9D1/viGlv\+WrNpWtd1XkRzyrFs=
AllowedIPs = 172\.16\.1\.2/32

